test/Api.IntegrationTest/SecretsManager/Controllers/AccessPoliciesControllerTests.cs

```diff
@@ -56,12 +56,16 @@ private async Task LoginAsync(string email)
     }
 
     [Theory]
-    [InlineData(false, false)]
-    [InlineData(true, false)]
-    [InlineData(false, true)]
-    public async Task CreateProjectAccessPolicies_SmNotEnabled_NotFound(bool useSecrets, bool accessSecrets)
+    [InlineData(false, false, false)]
```

Comment:
For these tests we want one of the flags to be false because we are testing for the scenario where it fails because of one being false, so we don't want all true's passed in 😄